Sources/TensorFlow/Operators/Basic.swift

```diff
@@ -1239,3 +1239,25 @@ extension Tensor {
         return areAxesInRange(axes.scalars)
     }
 }
+
+/// Checks to ensure elements within an array are distinct.
+///
```

Comment:
I think there's a precondition missing here, that the array is nonempty.  IIUC, your implementation will trap if it is violated.